backend/src/tests/account.test.ts

```diff
@@ -37,3 +37,55 @@ test('Login to account', async () => {
 
   expect(res.data.message).toBe("verify email");
 });
+
+test('Forgot password', async () => {
+  let randNum = Math.ceil(Math.random() * 99999);
+
+  let res = await axios.post(`${url}/api/account/forgotpassword`, {
+    email: `namejeff${randNum}@mailinator.com`
+  });
+
+  expect(res.data.message).toBe("success");
+});
+
+test('Change password', async () => {
+  let randNum = Math.ceil(Math.random() * 99999);
+
+  let res = await axios.post(`${url}/api/account/changepassword`, {
+    password: "newpassword",
+    token: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2MDdhMmY5NmVjOGU5ZTAwMmJlNGMxMTIiLCJmaXJzdG5hbWUiOiJTaGFseW5uIiwidXNlcm5hbWUiOiJNZXJjYWRvIiwiaWF0IjoxNjE4NjIwNDgxfQ.89a-kWSz5fr587osiGQ7kroabDzsxkpA87s-CvRmpLM"
```

Comment:
Dont use a pre-made token. This won't always exist. Create a user and login to get the token instead. Maybe you can use the token created in the login test.